transports/httpclient/src/main/java/org/commonjava/maven/galley/transport/htcli/model/HttpLocation.java

```diff
@@ -39,4 +39,5 @@
 
     int getProxyPort();
 
+    boolean isHostnameVerifier();
```

Comment:
Can we rename this to `isHostnameVerified()`? It'll improve readability / self-documentation.

Reply:
Sure. Updated.